OPHD/States/MapViewState.cpp

```diff
@@ -411,27 +411,34 @@ void MapViewState::onKeyDown(NAS2D::EventHandler::KeyCode key, NAS2D::EventHandl
 		mReportsUiSignal();
 		return;
 	}
-
+	int repeat = 1;
+	if ((uint16_t)mod & (uint16_t)NAS2D::EventHandler::KeyModifier::Shift)
+		repeat = 5;
 	switch(key)
 	{
+		
 		case NAS2D::EventHandler::KeyCode::KEY_w:
 		case NAS2D::EventHandler::KeyCode::KEY_UP:
-			mMapView->moveView(MapOffsetNorthWest);
+			for (int i = 0; i < repeat; i++)
+				mMapView->moveView(MapOffsetNorthWest);
```

Comment:
You can probably do this without the loop. If you check the definition, you'll find:
```cpp
constexpr auto MapOffsetNorthWest = MapOffset{DirectionNorthWest, 0};
```

Following definitions further, you'll find:
```cpp
constexpr auto DirectionNorthWest = NAS2D::Vector{-1, -1};
```

The `NAS2D::Vector` type has an overloaded `operator*`. Hence you could write something like:
```cpp
mMapView->moveView(MapOffset{DirectionNorthWest * repeat, 0});
```

Of course, at that point it may make sense to rename `repeat` to something else. Maybe `amount`, or `scale` or something similar.